blackbox.py

```diff
@@ -116,7 +117,7 @@ def cubetobox(x):
         str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")) + ' ...')
 
         # sampling next batch of points
-        fit = rbf(points)
+        fit = rbf(*np.transpose(points), function='cubic')

```

Comment:
New rbf() function requires another input format. The input list is transposed and then unpacked with *.

Old: [ [ x1, x2, ... , xd, val ] , [ ... ] , ... ]
New: [ x1, ... ] , [ x2, ... ] , ... , [ xd, ... ] , [ val, ... ]

Original function included cubic basis function, so now it's called manually in arguments.